### DISCUSSION TITLE
Chart having only podAntiAffinity and not podAffinity

### DISCUSSION THREAD

**Discussion by Bozz95** [argoproj/argo-helm#2274] [Q&A]
Hello,

I'm using this chart to deploy Argocd in an EKS cluster.
I noticed that there was a lot of traffic between the application controller and redis, which was between 2 different Availability Zones.

This spike in traffic coincides with a spike in network cost.

An idea to solve this cost was to make it so every Argocd pod was on the same node, but the chart only support podAntiAffinity.

How can I reach the solution of having all pods on the same node or Availability zone, or at least the application controller and redis (the culprits in these traffic spikes)?

**Comment by FernandoMiguel** [comment]
this would be great to make sure it is in the same AZ as possible
